### PR TITLE
Commit prep-html

### PR DESCRIPTION
**Pull request by bryan-jun**
https://user-images.githubusercontent.com/61750728/132963167-903ef713-4411-430f-8d98-31a9d0f60cad.mov

### CODE REVIEW

File: prep-html-links/contact-us/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="contact-us.html">Click here to go to the contact page</a></div>
+</body>
+</html>
```
Comment (scbowler) [inline prep-html-links/contact-us/index.html]: The `index.html` file should be outside of the `contact-us` folder.

File: prep-html-links/contact-us/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="contact-us.html">Click here to go to the contact page</a></div>
```
Comment (scbowler) [inline prep-html-links/contact-us/index.html]: Once the file has been moved you will need to update the `href` of your anchor tag

File: prep-html-links/contact-us/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div><a href="index.html">Click here to go to the home page</a></div>
```
Comment (scbowler) [inline prep-html-links/contact-us/contact-us.html]: Once the `index.html` file has been moved you will need to update the `href` of this anchor tag

**Review by scbowler — CHANGES_REQUESTED** [review]
Good start but some updates needed. Let me know if you have any questions!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Comment by bryan-jun** [comment]
https://user-images.githubusercontent.com/61750728/133145995-d6d3f51a-5f2f-484e-bea9-6c726bf37793.mov

**Pull request merged**